client/src/components/UnassignedMateCard/UnassignedMateCard.css

```diff
@@ -0,0 +1,4 @@
+.ant-card-actions {
```

Comment:
`UnassignedMateCard` is almost duplicated from `NewMateCard`. However, there are some nuances that are different and would require lots of refactoring. Since we were going to refactor these cards later when everything is more fleshed out, going this route for now.